src/test/java/com/ververica/flink/table/gateway/operation/ExplainOperationTest.java

```diff
@@ -74,11 +69,18 @@ public void testDescribe() {
 			"\t\texchange_mode : PIPELINED\n" +
 			"\t\tPartitioning : RANDOM_PARTITIONED\n" +
 			"\n";
-		ResultSet expected = ResultSet.builder()
-			.resultKind(ResultKind.SUCCESS_WITH_CONTENT)
-			.columns(ColumnInfo.create(ConstantNames.EXPLANATION, new VarCharType(false, expectedExplain.length())))
-			.data(Row.of(expectedExplain))
-			.build();
-		assertEquals(expected, resultSet);
+		compareResult(expectedExplain, resultSet);
```

Comment:
rename `testDescribe` to `testExplain`. It's better move the expected explain result to a resource file which is more friendly to reading